docs/user_guide/tutorials/02-collaboration/index.md

```diff
@@ -0,0 +1,230 @@
+
+```{exercise}
+:label: add-annotations
+
+- Add an annotation to your GIS file. Then add a comment to the annotation.
+- Ask your colleague to locate the annotation and add a reply.
+- Locate your colleague's reply from the right sidebar.
+
+```
+
+```{solution} add-annotations
+:class: dropdown
+
+
+- Right click on the desired location on your map.
+- Choose **Add Annotation**, click on the annotation and enter your comment.
+- Your colleague can see your annotation instantly; they can click it and reply.
+- Open the annotations panel on the right sidebar to view their reply.
+
+```
+
+---
+
+## 5. Collaborating on Notebooks
+
+Real-time collaboration in notebooks is a powerful tool for teams working on code together. It enables multiple users to write, edit, and run code simultaneously. This feature is ideal for live coding sessions, debugging, and data analysis projects.
+
+### 5.1. Accessing a Shared Notebook
+
+- To create a notebook, click on the **+** icon to open the Launcher, then select one of the kernels under **Notebook**.
+
+  ![Notebook](images/create_notebook.png)
+
+- Once a notebook is created, it is automatically accessible to all collaborators—no additional sharing is needed.
+- To open a shared notebook, click on the explorer button in the left sidebar, then locate and click on the notebook.
+
+  ![Notebook](images/open_notebook.png)
+
+- Anyone in the session can open, edit, and run the notebook.
+
+### 5.2. Real-Time Code Collaboration
+
+- As you write or execute code, every change is visible to your team instantly.
+- Multiple users can write, edit, and run code in the same notebook at the same time for a dynamic, interactive coding experience.
+
+![Notebook](images/notebook.gif)
+
+```{exercise}
+:label: notebook-collaboration
+
+- Create a new notebook and load your GIS document.
+- Ask your colleague to open the notebook, write the code to remove the air quality layer of the GIS file and run the code cell.
+
+```
+
+````{solution} notebook-collaboration
+:class: dropdown
+
+- Create a new notebook from the JupyterLab launcher (select Python kernel).
+- Load your GIS document using the following Python code:
+
+  ```python
+  from jupytergis import GISDocument
+  doc = GISDocument("your_project_name.jGIS")
+  ```
+- Your colleague can add and execute the following code to list all layers:
+  ```python
+  doc.layers
+  ```
+- Then they can find the air quality layer ID (the layer with the name Custom GeoJSON Layer) and remove it using:
+  ```python
+  air_quality_layer_id = "your_layer_id"
+  doc.remove_layer(air_quality_layer_id)
+  ```
+
+---
+
+Congratulations! You have completed the Collaboration Features of JupyerGIS tutorial. You now have the knowledge and tools to collaborate effectively with your team on GIS files and notebooks.
+
+## Additional resources
+
+- [JupyterGIS Blog Post](https://blog.jupyter.org/real-time-collaboration-and-collaborative-editing-for-gis-workflows-with-jupyter-and-qgis-d25dbe2832a6)
```

Comment:
I'm not sure how I feel about including this in "additional resources" as opposed to working it into prose. It feels like we should explain why we're linking this. What about...?


```suggestion

If you'd like to explore more of JupyterGIS' functionality, please view our
[JupyterGIS announcement blog post](https://blog.jupyter.org/real-time-collaboration-and-collaborative-editing-for-gis-workflows-with-jupyter-and-qgis-d25dbe2832a6)
for video demos of more features.
```

:shrug: